src/platforms/common/configuration/sampling.mdx

```diff
@@ -89,7 +94,12 @@ Whatever a transaction's sampling decision, that decision will be passed to its
 
 If the transaction currently being created is one of those subsequent transactions (in other words, if it has a parent transaction), the upstream (parent) sampling decision will always be included in the sampling context data, so that your <PlatformIdentifier name="traces-sampler" /> can choose whether and when to inherit that decision. (In most cases, inheritance is the right choice, to avoid partial traces.)
 
+<PlatformSection supported={["apple"]}>
+In Sentry SDK for Apple, for convenience, the <PlatformIdentifier name="traces-sampler" /> function can return a SentrySampleDecision, so that a parent's decision can be returned directly if that's the desired behavior.
+</PlatformSection>
+<PlatformSection notSupported={["apple"]}>
 In some SDKs, for convenience, the <PlatformIdentifier name="traces-sampler" /> function can return a boolean, so that a parent's decision can be returned directly if that's the desired behavior.
+</PlatformSection>
```

Comment:
```suggestion
In some SDKs, for convenience, the <PlatformIdentifier name="traces-sampler" /> function can return a boolean, so that a parent's decision can be returned directly if that's the desired behavior.
```